src/login/logind.conf.in

```diff
@@ -42,6 +42,8 @@
 #IdleAction=ignore
 #IdleActionSec=30min
 #RuntimeDirectorySize=10%
+# Unlike the other settings, 400k for RuntimeDirectoryInodesMax= below is not
+# the default value. Defaults to RuntimeDirectorySize= divided by 4096.
 #RuntimeDirectoryInodesMax=400k
```

Comment:
i think we should not mention any "400k" here if it's not actually used anywhre. just leave the rhs of the assignment empty

Reply:
Done.